src/Table/Table.tsx

```diff
@@ -228,7 +230,9 @@ const ActionColumn: React.FunctionComponent<ActionColumnProps> = (props: ActionC
                 <button
                     id={btnPrimaryRandomIds}
                     type="button"
-                    className="btn btn-outline-primary btn-sm"
+                    className={`btn btn-${props.primaryActionButton.buttonTheme ? props.primaryActionButton.buttonTheme : "outline-primary"} btn-${
+                        props.primaryActionButton?.buttonSize ? props.primaryActionButton?.buttonSize : "sm"
```

Comment:
the `buttonSize` is defined s `"btn-lg" | "btn-md" | "btn-sm"` but if I pass `"btn-lg"` as the `buttonSize` prop the className will result in`"btn-btn-lg"`